apps/web/src/pages/ApplicantDashboardPage/ApplicantDashboardPage.tsx

```diff
@@ -2,76 +2,108 @@
-    <>
-      <SEO title={""} description={""} />
-      <Hero
-        title={intl.formatMessage(
-          {
-            defaultMessage:
-              "Welcome back<hidden> to your applicant dashboard</hidden>, {name}",
-            id: "bw4CAS",
-            description:
-              "Title for applicant dashboard on the talent cloud admin portal.",
-          },
-          {
-            name: currentUser
-              ? getFullNameHtml(
-                  currentUser.firstName,
-                  currentUser.lastName,
-                  intl,
-                )
-              : intl.formatMessage(commonMessages.notAvailable),
-          },
-        )}
-        subtitle={""}
-      />
-    </>
-  );
-};
+import ReviewApplicationPreviewList, {
+  ReviewApplicationPreviewList_Fragment,
+} from "./components/ReviewApplicationPreviewList";
+import ReviewRecruitmentProcessPreviewList, {
+  ReviewRecruitmentProcessPreviewList_Fragment,
+} from "./components/ReviewRecruitmentProcessPreviewList";
 
 const ApplicantDashboard_Query = graphql(/* GraphQL */ `
-  query ApplicantDashboard_Query {
+  query ApplicantDashboard {
     me {
       id
       firstName
       lastName
+      poolCandidates {
+        ...ReviewApplicationPreviewList
+        ...ReviewRecruitmentProcessPreviewList
+      }
     }
   }
 `);
 
-export const ApplicantDashboardPageApi = () => {
+export const DashboardPage = () => {
+  const intl = useIntl();
+
   const [{ data, fetching, error }] = useQuery({
     query: ApplicantDashboard_Query,
   });
 
+  const user = data?.me;
+
+  const applications = getFragment<ReviewApplicationPreviewListFragment>(
+    ReviewApplicationPreviewList_Fragment,
+    unpackMaybes(user?.poolCandidates),
+  );
+
+  const recruitmentProcesses =
+    getFragment<ReviewRecruitmentProcessPreviewListFragment>(
+      ReviewRecruitmentProcessPreviewList_Fragment,
+      unpackMaybes(user?.poolCandidates),
+    ).filter(
+      (recruitmentProcess) =>
+        recruitmentProcess.finalDecisionAt &&
+        isQualifiedFinalDecision(recruitmentProcess.finalDecision?.value),
+    ); // filter for qualified recruitment processes
+
   return (
     <Pending fetching={fetching} error={error}>
-      <DashboardPage currentUser={data?.me} />
+      {!user ? (
+        <ThrowNotFound />
+      ) : (
+        <>
+          <SEO title={""} description={""} />
+          <Hero
+            title={intl.formatMessage(
+              {
+                defaultMessage:
+                  "Welcome back<hidden> to your applicant dashboard</hidden>, {name}",
+                id: "bw4CAS",
+                description:
+                  "Title for applicant dashboard on the talent cloud admin portal.",
+              },
+              {
+                name: user
+                  ? getFullNameHtml(user.firstName, user.lastName, intl)
+                  : intl.formatMessage(commonMessages.notAvailable),
+              },
+            )}
+            subtitle={""}
```

Comment:
subtitle is an optional prop so it can be omitted.
```suggestion
```

Reply:
Removed in [fa32530](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/fa325300416b98b5bd79f99d42db3d568cdf158a)